AnalysisStep/test/Ntuplizers/ZNtupleMaker.cc

```diff
@@ -268,8 +268,15 @@ namespace {
   std::vector<float> PhotonPt;
   std::vector<float> PhotonEta;
   std::vector<float> PhotonPhi;
+  std::vector<float> PhotonEnergyPostCorr ;
```

Comment:
For which studies do you need to have photons attached to Z trees, besides to the ZZ ones? This can potentially have a large impact on the total size of a production